types/lunr/index.d.ts

```diff
@@ -754,15 +757,31 @@ declare namespace lunr {
      * This helps to reduce the space used for storing the token set.
      */
     class TokenSet {
-        constructor()
+        constructor();
+
+        id: number;
+        final: boolean;
+
+        /**
+         * Key is a single character, the letter that represents the
```

Comment:
```suggestion
		 * @package
         * Key is a single character, the letter that represents the
```

Reply:
What are the `@package` comments for?

Reply:
I dunno either, but happy to add them.

Reply:
The whitespace formatting is kind of all over the place now (I think there’s a mix of tabs and spaces), and I want to know what these tags are for—@peterblazejewicz?